docs/architecture/adr/ADR032-sre-permissions.md

```diff
@@ -0,0 +1,28 @@
+# ADRXXX: Title
+
+## Status
+
+Accepted
+
+## Context
+
+As mitigation for some risks raised in threat modelling it was agreed that day-to-day access to the cluster was to be read-only for everyone. Only the concourse running in the cluster could make changes that originated from Github, which required several approvals before merging.
+
+Following the gradual rollout of several applications onto the GSP it became clear there were issues with the deployment procedures. This caused conflicting and contending pods to attempt to execute, resulting in application failures and deployment pipeline blockages. This was happening up to several times a day, depending on the level of activity. The remedial procedure involves escalating one or more members to cluster admin to allow the resources to be deleted, before revoking the admin permissions again. This could result in hours of wasted time for each occurrence.
```

Comment:
"This process requires 3 people to perform and could result in hours of wasted time for each occurrence."?